src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -75,7 +75,10 @@ impl CloudHypervisorInner {
 
     async fn handle_add_device(&mut self, device: DeviceType) -> Result<()> {
         match device {
-            DeviceType::ShareFs(sharefs) => self.handle_share_fs_device(sharefs.config).await,
+            DeviceType::ShareFs(sharefs) => {
+                // It's safe to unwrap as device_config is always there.
+                self.handle_share_fs_device(sharefs.config).await
```

Comment:
@Apokleos  Is the comment relevant here? I dont see we are unwrapping here. Config isnt an `Option` value as far i can see.

Reply:
Thx @amshinde for pointing out the issue. I'll fix it today.